Todo .go

```diff
@@ -176,14 +169,14 @@ func GetAllTodosHelper() []Todo {
 	for _, str := range data {
 
 		var todo Todo
-		todo.Id, _ = strconv.Atoi(str[0])        // converts type  string to int
+		todo.Id = str[0]                         // converts type  string to int
 		todo.Name = str[1]                       //noooo
 		todo.Done, _ = strconv.ParseBool(str[2]) // returns the bool value represented by string
 		todaArr = append(todaArr, todo)
 	}
 
 	// converts all the multi-dimensional array contents to struct and returns it
-	return todaArr
+	return todaArr[1:]
```

Comment:
if you return the second (and third, ...) elements, you can skip it on the other returns